src/App.jsx

```diff
@@ -14,70 +16,77 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
+export const App = () => {
+  const [good, setGood] = useState('Jam');
+  const noGoodSelectedBlock = (
     <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+  );
 
+  const goodSelectedBlock = (
     <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+      {good}
+      {' is selected'}
 
       <button
         data-cy="ClearButton"
         type="button"
         className="delete ml-3"
+        onClick={() => {
+          setGood('');
+        }}
```

Comment:
1. opening brackets in this case is not required
2. more correctly create separate function instead of providing anonymous function in onClick event
```suggestion
        onClick={() => setGood('')}
```